docs/sdk/sfa/sfa-flutter/usage.mdx

```diff
@@ -41,13 +41,12 @@ method. The method accepts `LoginParams`, and returns `SFAKey`.
 >
 
 <TabItem value="table">
-
-| Parameter            | Description                                                                                              |
-| -------------------- | -------------------------------------------------------------------------------------------------------- |
-| `verifier`           | The verifier obtained from the Web3Auth Dashboard. It's a mandatory field and takes `String` as a value. |
-| `verifierId`         | The verifierId used for the verification. It takes `String` as a value.                                  |
-| `idToken`            | The idToken of the user obtained from login provider. It takes `String` as a value.                      |
-| `aggregateVerifier?` | The aggregate verifier obtained from the Web3Auth Dashboard. It takes `String` as a value.               |
+| Parameter            | Description                                                                                                                                                                                                                |
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `verifier`           | The `verifier` parameter takes the name of the ustom verifier from the Web3Auth Dashboard. This is a required field that must be a `String`. If you're using an aggregate verifier, make sure to pass the sub-verifier name.  |
+| `verifierId`         | The `verifierID` takes the JWT verifier ID to be used for JWT/ID token verification. It can be an email, sub, or custom value available in the JWT token.                                                                           |
+| `idToken`            | The `idToken` accepts a JWT token obtained from the user's login provider.                                                                                                                                                 |
+| `aggregateVerifier?` | The `aggregateVerifier` parameter takes the name of the Aggregate verifier from the Web3Auth Dashboard.                                                                                                                        |
```

Comment:
SFA Android has `subVerifierInfoArray?` and SFA Flutter `aggregateVerifier`, and it shouldn't be the name of the aggregate verifier, if that's the case, we need to align all our SFA SDKs like this.

Reply:
Will be fixed in the next release.